deploy-argocd.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash -e
+
+# Deploy Applications to ArgoCD on EKS Cluster
```

Comment:
The description is misleading and needs to be updated.
The script does not deploy ArgoCD as the Terraform configuration takes care of that.

Reply:
so why do we need this script?